core/builder/src/main/java/io/quarkus/builder/BuildChainBuilder.java

```diff
@@ -30,7 +30,7 @@
  */
 public final class BuildChainBuilder {
 
-    private static final String GRAPH_OUTPUT = System.getProperty("jboss.builder.graph-output");
+    private static final String GRAPH_OUTPUT = System.getProperty("quarkus.builder.graph-output");
```

Comment:
Won't this cause a warning at startup about an unknown property?

Reply:
No, because we are not using the Smallrye Config API

Reply:
Ah sorry, you mean when running the application, you're right. I see it with the `quarkus.builder.log-conflict-cause` below too:
```
2023-08-10 08:41:07,076 WARN  [io.qua.config] (main) Unrecognized configuration key "quarkus.builder.graph-output" was provided; it will be ignored; verify that the dependency extension for this configuration is set or that you did not make a typo
```

Reply:
I've added a `BuilderConfig` to fix that. Thanks for catching this!